src/test/java/org/itsallcode/io/TestCapturingOuputStream.java

```diff
@@ -55,4 +70,39 @@ void testWithBufferedOutputStreamOnTopOfSystemOut() throws IOException
             System.setOut(previousOut);
         }
     }
+
+    @Test
+    void testOutputForwardedWhenCapturing() throws IOException
+    {
+        final OutputStream delegate = mock(OutputStream.class);
```

Comment:
If your reduce the code duplication between this and the next method, the difference might be easier to spot.

Reply:
✅